update/src/main.rs

```diff
@@ -340,6 +340,8 @@ fn write_impls(out: &mut String, typ: &'static str, builtins: Vec<String>) {
     writeln!(out, "\nimpl super::{typ} {{").unwrap();
 
     for thing in builtins {
+        let thing = thing.replace("-", "_"); // const value cannot have hyphens
```

Comment:
Otherwise I get this error

```
 cargo run --manifest-path update/Cargo.toml
    Finished dev [unoptimized + debuginfo] target(s) in 0.00s
     Running `update/target/debug/update`
error: expected one of `:`, `;`, or `=`, found `-`
    --> /private/tmp/cfg-expr/src/targets/builtins.rs:2734:14
     |
2722 | impl super::Abi {
     |                 - while parsing this item list starting here
...
2734 | pub const vec-extabi: Abi = Abi::new_const("vec-extabi");
     |              ^ expected one of `:`, `;`, or `=`
2735 | pub const x32: Abi = Abi::new_const("x32");
2736 | }
     | - the item list ends here

error: missing type for `const` item
    --> /private/tmp/cfg-expr/src/targets/builtins.rs:2734:14
     |
2734 | pub const vec-extabi: Abi = Abi::new_const("vec-extabi");
     |              ^ help: provide a type for the item: `: <type>`

error: failed to successfully format: exit status: 1

```